### PR TITLE
feat: #6154 delete pipeline

### PR DESCRIPTION
**Pull request by bashleigh**
# Pull request checklist

**Detail as per issue below (required):**

fixes: #6154

### CODE REVIEW

File: packages/deployment-service/src/functions/pipeline/pipeline-tear-down.ts [hunk 1]
```diff
@@ -67,14 +67,15 @@ const disableCloudFront = async (Id: string): Promise<any> => {
...
     s3Client.deleteObject(
       {
         Bucket: process.env.DEPLOYMENT_LIVE_BUCKET_NAME as string,
         Key: location,
       },
-      (error) => {
-        error ? reject(error) : resolve()
+      () => {
+        resolve()
+        // error ? reject(error) : resolve()
```
Comment (joshbalfour) [inline packages/deployment-service/src/functions/pipeline/pipeline-tear-down.ts:78]: idk if u wanna handle the error case here(?)

**Review by joshbalfour — APPROVED** [review]
LGTM 👍

**Review by willmcvay — APPROVED** [review]
LGTM

File: packages/deployment-service/src/functions/pipeline/pipeline-tear-down.ts (hunk 1 above)
Comment (bashleigh) [reply packages/deployment-service/src/functions/pipeline/pipeline-tear-down.ts:78]: yeaaaa not sure I wanna keep it. I'll try and get around it and see if it was just distro issues

**Pull request merged**